### ISSUE TITLE
Chore: spec repo docs updates and additions

### ISSUE THREAD

**Issue by scottaohara** [w3c/html-aria#354]
Some thoughts:

- [ ] expand the readme.  for instance, mention the conformance tools we check against.  Also, request that conformance tools that we are unaware of that use this spec inform us.
- [x] additional process for PRs / merging PRs.  e.g., we need to file issues with the conformance checkers and get at least 2 commitments to implement / implementations before we can merge into the spec. 
- [x] setup issue and PR templates to assist with people filing issues, or creating PRs and needing to make sure we get the conformance checker listing
- [x] cleanout old files that aren't used in the spec or for tests
- [ ] other....

cc @stevefaulkner, @patrickhlauke
